exchanges/coinbasepro/coinbasepro.go

```diff
@@ -766,9 +766,13 @@ func (c *CoinbasePro) SendAuthenticatedHTTPRequest(ctx context.Context, ep excha
 		n := strconv.FormatInt(time.Now().Unix(), 10)
 		message := n + method + "/" + path + string(payload)
 
+		secret, err := crypto.Base64Decode(creds.Secret)
+		if err != nil {
+			return nil, err
```

Comment:
I'd wrap this error personnally.
Because it'll be unclear it was a creds Secret error if it's allowed out naked.

Reply:
I've wrapped the error, couldn't find a typed error but I might have missed something.